utils/publish-legacy.sh

```diff
@@ -19,6 +23,6 @@ for tag in $tags; do
     npm_tag="--tag canary"
   fi
 
-  echo "Running \`npm publish $npm_tag\` in \"$(pwd)\""
-  npm publish $npm_tag
+  echo "Running \`npm publish --registry=https://registry.npmjs.com $npm_tag\` in \"$(pwd)\""
+  echo "DRY: npm publish --registry=https://registry.npmjs.com $npm_tag"
```

Comment:
Using `--registry` here is the _real_ fix. However, I've commented it out for now until we support `@vercel/static` and `@vercel/build-utils` in production.